### PR TITLE
Card #11

### PR DESCRIPTION
**Pull request by sayalis-cuelogic**
@onkartibe @aloka-cuelogic 

card link : https://trello.com/c/cS9UmKEo

Completed search recent posts feature.

### CODE REVIEW

**Comment by sayalis-cuelogic** [comment]
Card #8 , Card link : https://trello.com/c/fKvAbAIj
card #10 , Card link : https://trello.com/c/t8DY9w7Z
card #11, Card Link : https://trello.com/c/cS9UmKEo

Completed functionality.
Resolved conflicts.

File: post/models.py [hunk 1]
```diff
@@ -6,7 +6,7 @@ class PostInfo(models.Model):
     user_id = models.ForeignKey(User)
     post_name = models.CharField(max_length=100)
     author = models.CharField(max_length=100)
-    post_date = models.DateTimeField(auto_now_add=True,
+    post_date = models.DateField(auto_now_add=True,
```
Comment (aloka-cuelogic) [inline post/models.py]: auto_now doesn't require blank and null attributes

File: post/static/post/js/filter.js [hunk 2]
```diff
@@ -1,22 +1,54 @@
-$( document ).ready(function() {
-	var currentTime = new Date()
-	var year = currentTime.getFullYear()
-	for (year; year >= 1900; year--) {
-		$("#yearfilter").append('<option>'+year+'</option>');
-	};
+$(document).ready(function() {
+    var currentTime = new Date()
+    var year = currentTime.getFullYear()
+    for (var year; year >= 1900; year--) {
```
Comment (aloka-cuelogic) [inline post/static/post/js/filter.js]: year is defined just on top y again initialization for same

File: post/views.py [hunk 3]
```diff
@@ -44,6 +49,25 @@ def post(self, request):
             month = monthlist[month]
             post_data ['postdata']= PostInfo.objects.filter(post_date__month=month,
                             post_is_published='true')
+        elif date:
+            datelist = date.split("-")
+            senddata = {"postdata":
```
Comment (aloka-cuelogic) [inline post/views.py]: change this name

File: post/static/post/js/filter.js [hunk 4]
```diff
@@ -1,22 +1,54 @@
...
-			$('#monthfilter').removeAttr('disabled');
-			$('#yearfilter').attr('disabled', 'false');
-		}
-	});
+    $("#datefilter").datepicker({
+        autoclose: true
+    });
+
+    $(".radio").click(function() {
+        data = $('input:radio:checked').attr("id");
```
Comment (aloka-cuelogic) [inline post/static/post/js/filter.js]: can we work around this?
use toggle or any other thing which reduces the code

**Comment by aloka-cuelogic** [comment]
Reviewed Code.

Merging PR.

**Pull request merged**